_includes/templates/coap-dtls/coap-client-dtls.md

```diff
@@ -3,7 +3,7 @@ Install the CoAP client with DTLS support on Linux by following the next steps:
 - step 1: clone libcoap git repo: 
 
 ```bash
-git clone https://github.com/obgm/libcoap --recursive
+git clone https://github.com/obgm/libcoap --recursive --depth 1
```

Comment:
@yhavrysh please double-check that this is working fine. All other examples should be ok.